ecommerce-demo/dashboardPlugin/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pluginUrl": "https://gdc-dev-dashboard-plugins.s3.amazonaws.com/internal/dev/dedo/foceasting-plugin/dp_gd_deepar_plugin.js",
```

Comment:
Please use the bucket for production instead. We should wait for ticket https://gooddata.atlassian.net/browse/PC-5190 to be done so that we can use a CloudFront url to load the plugin's script. We should not expose to end-users that we are hosting the script on a S3 bucket.

Also, should we rename `dp_gd_deepar_plugin.js` → `dp_gd_forecasting_plugin.js` instead? IMO, the file should contain the word `forecasting` in its name to somewhat self-describe what the plugin does.

Reply:
After I have the correct Production information, I'll update pluginUrl and pluginName.